File: halcyon-web/src/main/resources/static/js/service/quartz/jobLogsService.js
/**
 * author:Marshal
 * description:service层
 */
app.service("jobLogsService", function ($http) {
    //base crud
    this.query = function (pageNum, pageSize, condition) {
        return $http.post("../quartz/jobLogs/query?pageNum=" + pageNum + "&pageSize=" + pageSize, condition);
    }
});

File: halcyon-quartz/src/main/java/com/marshal/halcyon/quartz/service/JobLogsService.java
package com.marshal.halcyon.quartz.service;

import com.marshal.halcyon.quartz.entity.JobLogs;

import java.util.List;

public interface JobLogsService {

    List<JobLogs> queryJobLogs(JobLogs condition, int pageNum, int pageSize);

    void saveJobLog(JobLogs jobLogs);
}
